ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -8,23 +8,37 @@
 
    .. cli:option:: --check
 
-      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+      Run in 'check' mode.
```

Comment:
Rather than saying "Run in 'check' mode." I would explain here what it actually does.

Reply:
926e0c9c8f0920f64b22f8a7840432f97657b71c